dotnet/src/Connectors/VectorData.Abstractions/ConnectorSupport/VectorStoreRecordDataPropertyModel.cs

```diff
@@ -14,20 +14,20 @@ namespace Microsoft.Extensions.VectorData.ConnectorSupport;
 public class VectorStoreRecordDataPropertyModel(string modelName, Type type) : VectorStoreRecordPropertyModel(modelName, type)
 {
     /// <summary>
-    /// Gets or sets a value indicating whether this data property is filterable.
+    /// Gets or sets a value indicating whether this data property is indexed for filtering.
```

Comment:
nit (because indexing is also for ordering I guess?)

```suggestion
    /// Gets or sets a value indicating whether this data property is indexed.
```

Reply:
Makes sense, updating everywhere